axelrod/tournament.py

```diff
@@ -234,20 +244,18 @@ def _write_interactions_to_dict(self, results):
                     self.interactions_dict[index_pair] = [interaction]
                 self.num_interactions += 1
 
-    def _run_parallel(self, processes: int=2, progress_bar: bool = False
-                      ) -> bool:
+    def _run_parallel(self, processes: int=2) -> bool:
         """
         Run all matches in parallel
 
         Parameters
         ----------
 
-        progress_bar : bool
-            Whether or not to update the tournament progress bar
+        processes : int
+            How many processes to use.
         """
         # At first sight, it might seem simpler to use the multiprocessing Pool
```

Comment:
Interesting!

But, we support both 3.5 and 3.6, so I reckon that comment needs to stay there for now.